### DISCUSSION TITLE
Downward compatibility

### DISCUSSION THREAD

**Discussion by musicgreator** [apohl79/audiogridder#746] [Q&A]
Will projects created with current versions of AudioGridder be compatible with later releases? 

I wonder if it is advisable for me to use AudioGridder for important projects, and for creating templates yet.

**Comment by apohl79** [comment]
Yes. You can always go back and switch between beta and stable versions too.

**Reply by musicgreator** [reply]
I'm trying to load a session created with 1.1.1 after I installed 1.2.0 Beta 5. AG only loads the plugins on the first track in the session. All other instances of AG show the instanciated plugins grey. And it seems unstable with the Monitor Window popping up, showing red dots...

**Reply by apohl79** [reply]
Ok this should not happen. Can you  collect more info? https://audiogridder.com/bug-reports/

thanks

**Reply by musicgreator** [reply]
It actually seems to happen even when the project has been created with 1.2.0 Beta 5. So the latest Beta version doesn't work here. 1.1.1 works fine. 

Will try to do a proper report....
Windows 10
Reaper

**Reply by musicgreator** [reply]
I've just sent a bug report via wetransfer. Apparently AudioGridder can't handle loading too many instances. I've reproduced this issue with different plugins.

**Reply by apohl79** [reply]
Did not get anything via wetransfer.

**Reply by musicgreator** [reply]
Hope you received it. Please let me know, if you have further questions regarding this issue.

**Comment by musicgreator** [comment]
This time wetransfer worked.

Andreas Pohl ***@***.***> schrieb am Fr., 8. Okt. 2021, 11:36:

**Reply by apohl79** [reply]
yes I got them. the crashes seem to be related to a sandboxing bug that I fixed. I also saw some issues with ffmpeg opening the screen device, but nothing really related to backwards compat problems. I will publish another beta soon that has a bunch of bugfixes. hope that helps.

**Reply by musicgreator** [reply]
Sounds great! The downward compatibility was just a misinterpretation of the issue by me. The issue seems only related to the current Beta, and not a downward compatibility issue as I thought.

**Comment by musicgreator** [comment]
That crashing issue seems to be solved with 1.2.0 Beta 6.
However I did some testing, and with large projects, AudioGridder has still problems loading. It will keep most plugins light grey (offline?), and I can see in the taskmanager: the memory usage climbs up and always falls down at different points. Seems like AudioGridder is not able to fully load everything, perhaps do to a high memory demand by some plugins in high quantity. This wasn't an issue with 1.1.1  I suspect it's some sort of memory issue, only happening when many plugins get loaded.

**Reply by musicgreator** [reply]
I sent you another report for this.

**Reply by musicgreator** [reply]
I've experienced the same issue with version 1.1.1. Sessions, where AG takes more than 10GB or something can't load reliably. When the dedicated process per instance option is active in AG, these sessions can be loaded, but they seem to take more CPU, at least compared to using the same plugins natively in the DAW.

**Comment by jaacMusic** [comment]
I'm experiencing the same issues. Can't load a project I created with Beta  4. I also have to load the project first with AG server off and then run AG server if not it would crash the project while loading in Cakewalk (Windows).
